test/entity/activeUser.test.js

```diff
@@ -1,20 +0,0 @@
-import expect from 'expect';
-import { ActiveUserHelper } from 'src/entity';
-import { randomString } from 'src/utils';
-
-describe('ActiveUser', function() {
```

Comment:
@thomasconner can you add tests to replace these, or more generally, test the `setActiveUser` and `getActiveUser` APIs?